graylog2-server/src/main/java/org/graylog/events/rest/EventNotificationsResource.java

```diff
@@ -152,6 +157,48 @@ public void delete(@ApiParam(name = "notificationId") @PathParam("notificationId
         resourceHandler.delete(notificationId);
     }
 
+    @POST
+    @Timed
+    @Path("/{notificationId}/test")
```

Comment:
Is this resource actually be used?

Reply:
It is not, but I think we should have it because that's what the old alert test API supported.